### ISSUE TITLE
MSPTech Organization

### ISSUE THREAD

**Issue by NickLaMuro** [msp-tech/code-of-conduct#1]
Chris,

While you are a wonderful human being with I am sure a couple hundred amazing qualities that make you truly exceptional (can't think of any to mention at the moment), it might make sense create a MSPTech github organization for this CoC and other items to live.

Okay, now that the snark is out of the way...

I could think of a few things that might live under this org:
- This Code of Conduct
- Home page source code
- Community Chat bot (if that is really necessary)
- Etc.

I have no intention of requesting that I be an organizer with this request (I would actually prefer the opposite for the time being), I just think that the group will be more approachable to new comers if there is a "official" looking github organization attached to it.  It would also make the group a bit more findable and less of a "word of mouth" type group (though, if that was your intent, then maybe this isn't the best course of action).

**Comment by chrisarcand** [comment]
I completely agree. I kind of threw this up quickly because I wanted _something_ up before too many people joined. I will definitely do that very soon.

Re: Hubot: If people want one, we can certainly do that. Another piece that I thought I'd wait on for a bit. 

Thanks for your input :+1:

**Comment by NickLaMuro** [comment]
A hubot alternative:  https://www.lita.io/

Though, since this is a cross language organization, it might make sense to stick with hubot since it is probably the most widely used.  No preference really, just wanted an excuse to mess around with it.

`</2cents>`

**Comment by maxfierke** [comment]
:+1: CoC very important

Good guidelines for now:
1. Remember the human on the other side of the chat window
2. Assume good faith until there is clearly none
3. Probably other things that need to be specified

Now someone do lawyer stuff to that pls

**Comment by chrisarcand** [comment]
Organization created. https://github.com/msp-tech/code-of-conduct

Let's talk hubot a bit later in a separate issue.
